ui/app/components/clients/page/overview.ts

```diff
@@ -5,4 +5,4 @@
 
 import ActivityComponent from '../activity';
 
-export default ActivityComponent;
+export default class ClientsOverviewPageComponent extends ActivityComponent {}
```

Comment:
Originally, both this component and the new acme page component had `export default ActivityComponent;` because there are no additional getters in each. However, this caused the following error: 
```
Error: Cannot call `setComponentTemplate` multiple times on the same class (`ClientsActivityComponent`)
```

I can revert this change here, but it seemed reasonable that this component have a separately named backing class?